lib/ngx/balancer_stream.lua

```diff
@@ -0,0 +1,82 @@
+-- Copyright (C) Yichun Zhang (agentzh)
```

Comment:
Well I'll let you lay this out this however you like. I'll defer to your experience.